src/auth.ts

```diff
@@ -0,0 +1,61 @@
+/*
+auth.ts - Provides auth related methods for the Bluebutton class
+*/
+import crypto from "crypto";
+
+import BlueButton from ".";
+
+export type authData = {
```

Comment:
I generally see classes/types capitalized 
```suggestion
export type AuthData = {
```